vms/platformvm/camino_vm_test.go

```diff
@@ -476,7 +476,7 @@ func TestDepositsAutoUnlock(t *testing.T) {
 		Address: depositOwnerAddrBech32,
 	}})
 	vm.ctx.Lock.Lock()
-	defer func() { require.NoError(vm.Shutdown(context.Background())) }() //nolint:revive
+	defer func() { require.NoError(vm.Shutdown(context.Background())) }() //nolint:lint
```

Comment:
Sure? Reading the error I expected that the rule can be dropped completely (??)

Reply:
@mo-c4t

Reply:
after updating the golangci-lint version, the error comes with nolint:lint and not nolint:revive